tests/unit/node/test_node_snapshot.py

```diff
@@ -150,3 +150,27 @@ def _fun(self):
     progress = response.json()["progress"]
     assert progress["failed"]
     assert progress["final"]
+
+
+@pytest.mark.timeout(2)
+def test_snapshot_shrinked_file_too_small(snapshotter):
+    PATH = snapshotter.src / "shrinky1"
```

Comment:
Why is this uppercase? They are usually globals.

Reply:
parameterized the tests